modules/caddyhttp/fileserver/matcher.go

```diff
@@ -256,6 +270,26 @@ func strictFileExists(file string) bool {
 	return !stat.IsDir()
 }
 
+// firstSplit returns the first result where the path
+// can be split in two by a value in m.SplitPath. The
+// result is the first piece of the path that ends with
+// in the split value. Returns the path as-is if the
+// path cannot be split.
+func (m MatchFile) firstSplit(path string) string {
+	lowerPath := strings.ToLower(path)
+	for _, split := range m.SplitPath {
+		if idx := strings.Index(lowerPath, strings.ToLower(split)); idx > -1 {
+			pos := idx + len(split)
+			// skip the split if it's not the final part of the filename
+			if pos != len(path) && !strings.HasPrefix(path[pos:], "/") {
```

Comment:
Would it be simpler (and still correct?) if we just suffix each split with `/`?

Reply:
No because then `/remote.php` wouldn't work

Reply:
Good point